rust/db_handling/src/identities.rs

```diff
@@ -1362,16 +1363,10 @@ where
 
     let style = address_details.encryption.identicon_style();
     let address = Address {
-        base58: print_multisigner_as_base58_or_eth(
-            multisigner,
-            Some(network_specs.base58prefix),
-            address_details.encryption,
-        ),
```

Comment:
Why is address not the part of `Address`? It _literally_ the address, be it `ss58` or ethereum addr? I would suggest leaving it here but renaming to something address-format agnostic

Reply:
The address could probably be an enum with variants `ss58` and `eth`  to add a bit of structure

Reply:
> It literally the address

Not really. It's an address + encryption. 
`print_multisigner_as_base58_or_eth` depends on `network_specs`, but `AddressDetails` has a list of `NetworkSpecsKey`.
https://github.com/paritytech/parity-signer/blob/master/rust/definitions/src/users.rs#L39

So, in general, it's not possible to define `ss58` having only `AddressKey`

Reply:
Agreed. And we better have this in a separate PR.